tokio-console/src/config.rs

```diff
@@ -95,6 +96,23 @@ pub enum OptionalCmd {
     ///     $ tokio-console gen-config > console.toml
     ///
     GenConfig,
+
+    /// Generate shell completions
+    ///
+    /// The completion script will be written to stdout.
+    /// The completion script should be saved in the shell's completion directory.
+    /// This depends on which shell is in use.
+    ///
+    /// For example, using zsh:
+    ///
+    ///     $ tokio-console gen-completion zsh > ~/.zsh_functions/_tokio_console
```

Comment:
@hawkw 
Sorry, I'm mistaken.
`$FPATH` has multiple path values so it's not work `$FPATH/_tokio_console`

I worried about this sample, because `.zsh_functions` directory name. This name is my config value on .zshrc. I don't know common name.

Reply:
hmm, if this isn't going to work for everyone, maybe we should remove the sample for now, and add it back later if we can come up with one that will work universally?